assets/javascripts/discourse/widgets/discourse-reactions-actions.js.es6

```diff
@@ -122,21 +121,18 @@ export default createWidget("discourse-reactions-actions", {
       classes.push("has-reacted");
     }
 
-    if (attrs.post.current_user_used_main_reaction) {
+    if (post.current_user_used_main_reaction) {
       classes.push("has-used-main-reaction");
     }
 
     if (
-      attrs.post.likeAction &&
-      (attrs.post.likeAction.canToggle || attrs.post.likeAction.can_undo)
+      post.likeAction &&
+      (post.likeAction.canToggle || post.likeAction.can_undo)
     ) {
       classes.push("can-toggle-main-reaction");
     }
 
-    if (
-      attrs.post.reactions.length === 1 &&
-      attrs.post.reactions[0].id === mainReaction
-    ) {
+    if (post.reactions.length == 1 && post.reactions[0].id === mainReaction) {
```

Comment:
Are you sure you configured eslint/prettier correctly ? this == is not valid

Reply:
I didn't used eslint😅 ill use it from now😊